weave-js/src/components/Button/Button.tsx

```diff
@@ -4,153 +4,153 @@
  * https://www.figma.com/file/01KWBdMZg5QM9SRS1pQq0z/Design-System----Robot-Styles?type=design&node-id=5956-31813&mode=design&t=Gm4WWGWwgjdfUUTe-0
  */
 
-import {TooltipContentProps} from '@radix-ui/react-tooltip';
+import {type TooltipContentProps} from '@radix-ui/react-tooltip';
```

Comment:
Importing with `type` explicitly makes it easier to reason with b/c components and constructor functions both use PascalCase in their naming conventions. so without this, the imports are indistinguishable.